packages/format-library/src/default-formats.native.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import { bold } from './bold';
+import { code } from './code';
+import { italic } from './italic';
+import { link } from './link';
+import { strikethrough } from './strikethrough';
+
+export default [
```

Comment:
Why is this different from the web version?

Reply:
We're not supporting inline images for now, and I think trying to register `image` was throwing errors because of components we haven't ported yet.

Reply:
Can it not be filtered out in the RichText component?

Reply:
When I did this, it was triggering some errors on registration, but it seems fine now so I reverted this part